datacenter/ucp/2.1/guides/admin/backups-and-disaster-recovery.md

```diff
@@ -14,27 +14,50 @@ The next step is creating a backup policy and disaster recovery plan.
 ## Backup policy
 
 As part of your backup policy you should regularly create backups of UCP.
-To create a backup of UCP, use the `docker/ucp backup` command. This creates
-a tar archive with the contents of the [volumes used by UCP](../architecture.md)
-to persist data, and streams it to stdout.
 
-You need to run the backup command on a UCP manager node. Since UCP stores
-the same data on all manager nodes, you only need to create a backup of a
-single node.
+To create a UCP backup, you may use the `{{ page.docker_image }} backup` command
+against a single UCP manager, according to the instructions in the next section.
+This command creates a tar archive with the contents of all the [volumes used by
+UCP](../architecture.md) to persist data and streams it to stdout.
+
+You only need to run the backup command on a single UCP manager node. Since UCP
+stores the same data on all manager nodes, you do not need to capture periodic
+backups from more than one manager node.
 
 To create a consistent backup, the backup command temporarily stops the UCP
 containers running on the node where the backup is being performed. User
-containers and services are not affected by this.
+resources, such as services, containers and stacks are not affected by this
+operation and will continue operating as expected. Any long-lasting `exec`,
+`logs`, `events` or `attach` operations against the affected manager node will
```

Comment:
Internally we use the word "against" a lot but when reading things for the first time, that word is a bit ambiguous. Maybe

"If you're running commands like `exec`, `logs`, `events`, or `attach` on a manager node, when performing a backup those commands will be disconnected."